enterprise/server/remote_execution/executor/executor.go

```diff
@@ -204,10 +204,10 @@ func (s *Executor) ExecuteTaskAndStreamResults(ctx context.Context, st *repb.Sch
 	if err != nil {
 		return finishWithErrFn(status.WrapErrorf(err, "error creating runner for command"))
 	}
-	finishedCleanly := false
-	defer func() {
-		go s.runnerPool.TryRecycle(ctx, r, finishedCleanly)
-	}()
+	//finishedCleanly := false
+	//defer func() {
+	//	go s.runnerPool.TryRecycle(ctx, r, finishedCleanly)
+	//}()
```

Comment:
We always want to call `TryRecycle()` otherwise the VMs will accumulate and the executor will crash eventually.

What is your goal here? (I think if you want to test whether it happens on recycled runners, you could revert the changes to this file and let the runner be recycled normally)

Reply:
Ah I see that TryRecycle needs to be called for runner.finalize(). The goal is for the workflow to always be executed on a clean runner, so I don't want any runners to be recycled back into the pool